src/web/mod.rs

```diff
@@ -243,14 +243,11 @@ impl MatchSemver {
 /// underscores (`_`) and vice-versa. The return value will indicate whether the crate name has
 /// been matched exactly, or if there has been a "correction" in the name that matched instead.
 fn match_version(conn: &Connection, name: &str, version: Option<&str>) -> Option<MatchVersion> {
-    // version is an Option<&str> from router::Router::get
-    // need to decode first
```

Comment:
Why move this comment onto a single line?

Reply:
There's not really a reason for it to be two, it's one continuous thought